docs/conf.py

```diff
@@ -59,7 +59,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'ENCODE Utils'
+project = 'IGVF Utils'
 copyright = '2018, Nathaniel Watson'
 author = 'Nathaniel Watson'
 
```

Comment:
I am not sure how copyright of forked code works? Not saying we should remove Nathaniel, but maybe should say also Jennifer Jou?